### PR TITLE
feat(monorepo): use lerna for running tasks in parallel

### LINKED ISSUE #441: actually use lerna for tasks
as a developer
i want to run tasks using lerna
so that my tests and builds run in parallel, saving me time and making CI pipeline faster

dod: 
- all tasks are run using lerna, with correct scoping and parallelization

### PR DESCRIPTION
**Pull request by vacekj**
- use lerna for running top-level scripts
- update husky to use lerna
- update lerna version to v5
- use nx in lerna for faster task running

##### Refers/Fixes

fixes #441 

##### Testing

no change in tests

### CODE REVIEW

File: packages/round-manager/package.json [hunk 1]
```diff
@@ -61,7 +61,8 @@
     "lint": "eslint ./src --cache",
     "lint:fix": "eslint ./src --fix --cache",
     "prepare": "cd ../.. && husky install",
-    "lint-staged": "lint-staged"
+    "lint-staged": "lint-staged",
+    "slither": "slither ."
```
Comment (shavinac) [inline packages/round-manager/package.json:65]: do we have any solidity files in `round-manager` package? if not, then we don't need the `slither` script here

Comment (thelostone-mc) [reply packages/round-manager/package.json:65]: That's a good callout

Comment (thelostone-mc) [reply packages/round-manager/package.json:65]:
```suggestion
```

**Comment by thelostone-mc** [comment]
resolved the conflicts + addressed review comment

**Review by thelostone-mc — APPROVED** [review]

**Pull request merged**